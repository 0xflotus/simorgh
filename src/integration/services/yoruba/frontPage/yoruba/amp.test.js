/**
 * @service yoruba
 * @pathname /yoruba
 */

import runAmpTests from '../../../../pages/frontPage/ampTests';

describe('AMP yoruba frontPage', runAmpTests);
