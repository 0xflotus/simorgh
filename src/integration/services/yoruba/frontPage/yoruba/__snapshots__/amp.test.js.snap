// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP yoruba frontPage A11y I can see the skip to content link 1`] = `"Fò kọjá sí nnkan tí ó wà nínú rẹ̀"`;

exports[`AMP yoruba frontPage AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP yoruba frontPage AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP yoruba frontPage Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=107&p=yoruba.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:cps:5bc0e956-8797-3f64-e050-17ac2b8c7c23]&x2=[amp]&x3=[news-yoruba]&x4=[yo]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[index-home]&x8=[simorgh]&x9=[Àbáwọlé+-+BBC+News+Yorùbá]&x11=[1970-01-18T10:59:00.185Z]&x12=[1970-01-19T01:43:03.450Z]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP yoruba frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/yoruba"`;

exports[`AMP yoruba frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. BBC kò mọ̀ nípa àwọn ohun tí ó wà ní àwọn ojú òpó tí ó wà ní ìta. Ọwọ́ tí a fi mú ìbáṣepọ̀ ti ìta."`;

exports[`AMP yoruba frontPage Header I can see the branding 1`] = `"BBC News, Yorùbá"`;

exports[`AMP yoruba frontPage SEO Canonical link 1`] = `"http://localhost:7080/yoruba"`;

exports[`AMP yoruba frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP yoruba frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP yoruba frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP yoruba frontPage SEO Heading level 1 1`] = `"BBC News, Yorùbá - Ìròyìn"`;

exports[`AMP yoruba frontPage SEO Lang attribute 1`] = `"yo"`;

exports[`AMP yoruba frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/yoruba.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/yoruba",
    "@type": "WebPage",
    "name": "Àbáwọlé",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/yoruba.png",
      "width": 1024,
    },
    "name": "BBC News Yorùbá",
    "publishingPrinciples": "https://www.bbc.com/yoruba/institutional-48528718",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/yoruba.png",
  "url": "http://localhost:7080/yoruba",
}
`;

exports[`AMP yoruba frontPage SEO OG description 1`] = `"Ibí yìí lo ti máa rí ojúlówó ìròyìn àjáàbalẹ̀ lórí ojú ọjọ́, ètò ọrọ̀ ajé, ìdánilárayá, òṣèlú, ìròyìn àgbáyé ní ìṣẹ́jú kan àtàwọn míràn. BBC Yorùbá máa ń ṣe ìròyìn àkànṣe, fídíò, ọ̀rọ̀ ẹnu, fọ́tò tó jojú ní gbèsè àti itọ́ni onísọ̀rọ̀-ǹ-gbèsì tó dáńtọ́"`;

exports[`AMP yoruba frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/yoruba.png"`;

exports[`AMP yoruba frontPage SEO OG image alt 1`] = `"BBC News Yorùbá"`;

exports[`AMP yoruba frontPage SEO OG locale 1`] = `"yo"`;

exports[`AMP yoruba frontPage SEO OG site name 1`] = `"BBC News Yorùbá"`;

exports[`AMP yoruba frontPage SEO OG title 1`] = `"Àbáwọlé - BBC News Yorùbá"`;

exports[`AMP yoruba frontPage SEO OG type 1`] = `"website"`;

exports[`AMP yoruba frontPage SEO OG url 1`] = `"http://localhost:7080/yoruba"`;

exports[`AMP yoruba frontPage SEO Page title 1`] = `"Àbáwọlé - BBC News Yorùbá"`;

exports[`AMP yoruba frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP yoruba frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP yoruba frontPage SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`AMP yoruba frontPage SEO Twitter description 1`] = `"Ibí yìí lo ti máa rí ojúlówó ìròyìn àjáàbalẹ̀ lórí ojú ọjọ́, ètò ọrọ̀ ajé, ìdánilárayá, òṣèlú, ìròyìn àgbáyé ní ìṣẹ́jú kan àtàwọn míràn. BBC Yorùbá máa ń ṣe ìròyìn àkànṣe, fídíò, ọ̀rọ̀ ẹnu, fọ́tò tó jojú ní gbèsè àti itọ́ni onísọ̀rọ̀-ǹ-gbèsì tó dáńtọ́"`;

exports[`AMP yoruba frontPage SEO Twitter image alt 1`] = `"BBC News Yorùbá"`;

exports[`AMP yoruba frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/yoruba.png"`;

exports[`AMP yoruba frontPage SEO Twitter site 1`] = `"@BBCNews"`;

exports[`AMP yoruba frontPage SEO Twitter title 1`] = `"Àbáwọlé - BBC News Yorùbá"`;
